### PR TITLE
Fix broken docs link to ISO 8601 duration format

### PR DESCRIPTION
**Pull request by mvxt**
## Description
Very minor change to a broken external docs link to the ISO 8601 timestamp format.

## Testing plan

N/A - make sure new link works. See:

image

## Related links
- [Old, broken link](https://www.digi.com/resources/documentation/digidocs/90001437-13/reference/r_iso_8601_duration_format.htm)
- [New, fixed link](https://www.digi.com/resources/documentation/digidocs/90001488-13/reference/r_iso_8601_duration_format.htm)

## PR readiness 
- [x] Relevant documentation has been updated if this PR adds or updates attributes, resources, or data sources.
- [x] An entry has been added to the changelog, if necessary.
  - Does this require a changelog entry??

### CODE REVIEW

**Comment by elg0nz** [comment]
@mvxt no this does not need a changelog entry.

**Review by elg0nz — APPROVED** [review]
Note: this change won't be reflected until we publish the next version.
Given that this is a relatively small change, unless you really need to get it out, I would just wait until the next PR is merged. Thanks!

**Comment by mvxt** [comment]
@elg0nz No problem, it's not a big deal. Feel free to merge this PR when the right time comes

**Pull request merged**
